contracts/base/LoanTermsConsensus.sol

```diff
@@ -96,6 +96,7 @@ contract LoanTermsConsensus is LoanTermsConsensusInterface, Consensus {
         );
         borrowerToLastLoanTermRequest[request.borrower] = now;
 
+        // TODO: Remove redundant event
```

Comment:
because the only time these events are emitted are in the same transaction:
https://github.com/teller-protocol/teller-protocol-v1/blob/dd0704e9736a940154c45113b202d6842ad83310/contracts/base/LoansBase.sol#L244-L253